helm/cluster-vsphere/values.yaml

```diff
@@ -14,6 +14,7 @@ network:
   - "192.168.0.0/16"
   servicesCidrBlocks:
   - "192.168.1.0/16"
+  loadBalancersCidrBlocks: []
```

Comment:
@vxav @marians I need feedback on the name of this parameter. We can mimic https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/values.yaml#L69 here again. What do you think?

Reply:
That looks sensible to me

Reply:
I am ok with your proposal